About.md

```diff
@@ -0,0 +1,63 @@
+# About the Lattice Standard
+
+## What the Lattice Standard is
+
+1. The Lattice Standard is an *abstract* description of a lattice for the simulated transport of charged particles.
+
+1. A Lattice is an ordered list of regions in space distinguished by the presense of (possibly time-varying) specially described electromagnetic fields,
+materials, apertures and other possible engineered structures. For simplicity, we will denote these regions as _lattice_elements_ although a region
+may be distinguished by the absence of material (drift space.)
+
+1. A lattice will have the notion of a _reference_trajectory_ which is the defined path of the particle which the lattice is intended to transport.
```

Comment:
This is a bit confusing. Especially since the particles may never follow the reference trajectory (EG: in a wiggler the reference trajectory is straight but the actual particle path is oscillatory). I think better is:

A lattice will have the notion of a _reference_trajectory_, also called the zero orbit, about which the curvilinear coordinate system of the lattice is defined. This curvilinear coordinate system, called _machine coordinates_ in the standard, establishes a coordinate system which may be used to describe particle trajectories and lattice element alignment.

Reply:
Yes, I was unsure of how to deal with things like undulators and even what MAD calls RBENDS. We definitely need some language that can deal with it but it wasn't obvious to me without maybe a figure what your terminology refers to and how you would use it to place elements at positions.

Reply:
Yes I plan on submitting a PR that will go into this in detail.